### PR TITLE
Add image_url into Projects

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #119**

- [x] Add `image_url` table
- [x] Create migration
- [x] Insert data into the database

### CODE REVIEW

**Comment by Ovifer13** [comment]
@sofiaromorales I already did the migration and did a script to update the new fields
**Proof**
image
image

**Comment by sofiaromorales** [comment]
@Ovifer13 let's run down this migration and add the fields
```
tile_image_url
device_image_url
logo_image_url
background_banner_image_url
```

**Comment by Ovifer13** [comment]
@sofiaromorales changes done

File: pages/api/graphql/schema/types/ProjectType.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = gql`
     description: String
     client_name: String
     client_description: String
+    image_url: String
```
Comment (sofiaromorales) [inline pages/api/graphql/schema/types/ProjectType.ts:11]: @Ovifer13 pls update the project schema

**Review by sofiaromorales — CHANGES_REQUESTED** [review]

File: pages/api/graphql/schema/types/ProjectType.ts (hunk 1 above)
Comment (Ovifer13) [reply pages/api/graphql/schema/types/ProjectType.ts:11]: Sorry, totally forgot I had to update there too. Omw

**Comment by Ovifer13** [comment]
image

**Review by sofiaromorales — APPROVED** [review]

**Pull request merged**